style/elixir/README.md

```diff
@@ -0,0 +1,11 @@
+Elixir
+======
+
+[Sample](sample.ex)
+
+* Avoid macros.
+* Don't separate multiple declarations of the same function with a newline.
+* Indent pipes by two spaces when using the match operator (`=`).
+* Don't indent pipes when not using the match operator (`=`).
```

Comment:
I agree with the consistency part but I'd go the other way and say don't indent ever.